app/src/services/bucketPermission.js

```diff
@@ -108,6 +108,22 @@ const service = {
     }
   },
 
+  /**
+   * @function getBucketIdsWithObject
+   * Searches for specific (bucket) object permissions
+   * @param {string|string[]} [params.userId] Optional string or array of uuids representing the user
+   * @returns {Promise<object>} The result of running the find operation
+   */
+  getBucketIdsWithObject: async (userIds) => {
```

Comment:
`userIds` should be singular `userId`. While it can accept one or many, we generally stick with singular naming unless the parameter must always contain multiple values.